pkg/cmd/roachtest/test_registry.go

```diff
@@ -44,19 +44,50 @@ const (
 type OwnerMetadata struct {
 	SlackRoom    string
 	ContactEmail string
+	// TriageColumnID is the column id of the project column the team uses to
+	// triage issues. Unfortunately, there appears to be no way to retrieve this
+	// programmatically from the API.
+	//
+	// To find the triage column for a project, run the following curl command:
+	// curl -u yourusername:githubaccesstoken -H "Accept: application/vnd.githubinertia-preview+json" \
+	// https://api.github.com/repos/cockroachdb/cockroach/projects
+	//
+	// Then, for the project you care about, curl its columns URL, which looks
+	// like this:
+	// https://api.github.com/projects/3842382/columns
+	//
+	// Find the triage column you want, and pick its ID field.
+	TriageColumnID int
 }
 
 // roachtestOwners maps an owner in code (as specified on a roachtest spec) to
 // metadata used for github issue posting/slack rooms, etc.
 var roachtestOwners = map[Owner]OwnerMetadata{
-	OwnerAppDev:       {SlackRoom: `app-dev`, ContactEmail: `rafi@cockroachlabs.com`},
-	OwnerBulkIO:       {SlackRoom: `bulk-io`, ContactEmail: `david@cockroachlabs.com`},
-	OwnerCDC:          {SlackRoom: `cdc`, ContactEmail: `ajwerner@cockroachlabs.com`},
-	OwnerKV:           {SlackRoom: `kv`, ContactEmail: `andrei@cockroachlabs.com`},
-	OwnerPartitioning: {SlackRoom: `partitioning`, ContactEmail: `andrei@cockroachlabs.com`},
-	OwnerSQLExec:      {SlackRoom: `sql-execution-team`, ContactEmail: `alfonso@cockroachlabs.com`},
-	OwnerSQLSchema:    {SlackRoom: `sql-schema`, ContactEmail: `lucy@cockroachlabs.com`},
-	OwnerStorage:      {SlackRoom: `storage`, ContactEmail: `peter@cockroachlabs.com`},
+	OwnerAppDev: {SlackRoom: `app-dev`, ContactEmail: `rafi@cockroachlabs.com`,
+		TriageColumnID: 8532151,
```

Comment:
Nice jankiness. Maybe just add a comment next to each one naming the specific board/column.